data/ExpeditionAvatarEffect.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
Data for expedition avatar effect can be found in ProudSkillExcelConfigData, there is no need to hardcode avatar expedition effects one by one.

Reply:
Ah, yes, yes, yes, what you say is right. I really hope u or someone can fix the expedition,bye!